sales/js/app.js

```diff
@@ -149,22 +161,53 @@ function renderTableBody(parent, bodyElements, subArrayName){
   }
 }
 
-// Cookies table
-processLocationStats();
-var parentMyElement = document.getElementById('mylistsOfStats');
-var table = document.createElement('table');
-parentMyElement.appendChild(table);
+function renderTables(){
+  // Cookies table
+  processLocationStats();
+  var parentMyElement = document.getElementById('mylistsOfStats');
+  var table = document.createElement('table');
+  parentMyElement.appendChild(table);
 
-renderTableHeader(table, providingHeaders('Daily Location Total','Store Name'));
-renderTableBody(table, stores, 'renderStatsAsRow');
-renderTableFooter(table, providingSumRow());
+  renderTableHeader(table,
+    providingHeaders('Daily Location Total','Store Name'), 'Cookie Sales'
+  );
+  renderTableBody(table, stores, 'renderStatsAsRow');
+  renderTableFooter(table, providingSumRow('locationsStats', 'totalCookiesSold'));
+
+  // Tossers Table
```

Comment:
You only need one table. You can totally have this in there for more, but you only need one!